cspell.json

```diff
@@ -161,6 +161,7 @@
     "opencollective",
     "opensource",
     "opuuus",
+    "overlimit",
```

Comment:
I guess the spellchecker wants you to write it as `overLimit` instead of `overlimit` since these are two words.